src/pages/RestrictedAction/Workspace/WorkspaceOwnerRestrictedAction.tsx

```diff
@@ -30,7 +31,7 @@ function WorkspaceOwnerRestrictedAction() {
         >
             <HeaderWithBackButton
                 title={translate('workspace.restrictedAction.restricted')}
-                onBackButtonPress={Navigation.goBack}
+                onBackButtonPress={Navigation.resetToHome}
```

Comment:
And we can revert back to `Navigation.goBack` to prevent resetting the nav stack.

Reply:
Adjusted